src/foam/nanos/auth/User.js

```diff
@@ -904,8 +905,47 @@ foam.CLASS({
       javaCode: `
         HashMap<String, NotificationSetting> settingsMap = new HashMap<String, NotificationSetting>();
 
+        settingsMap = getImpliedNotificationSettings(x);
+        for ( NotificationSetting setting : settingsMap.values() ) {
+          System.err.println(setting.getClassInfo().getId());
```

Comment:
Assuming this was during debugging.  Please replace with logger if still required.
If you are doing this to avoid a loop on logging warning, error, which in turn generate notifications, please use the StdoutLogger.instance().